### DISCUSSION TITLE
Socket io on aws linux ec2 instance

### DISCUSSION THREAD

**Discussion by suryapolnati** [socketio/socket.io#4488] [Q&A]
Hello All,
I have followed this link [https://socket.io/docs/v4/pm2/](url) to create a socket io worker process on ec2 linux instance. However, I am not able to connect using a client running on my local machine.

here is my client code 
`socket = io('http://<public DNS name of ec2>:8080');` 

Kindly Help.

Thanks,
Surya

**Comment by darrachequesne** [comment]
Hi! Are you able to reach the Socket.IO server?

```
curl "<the server URL>/socket.io/?EIO=4&transport=polling"
```

Reference: https://socket.io/docs/v4/troubleshooting-connection-issues/#problem-the-socket-is-not-able-to-connect

**Comment by suryapolnati** [comment]
Hello,
Thankyou for taking time and replying!.

I have ran the command and i get the below error.
![image](https://user-images.githubusercontent.com/59924040/194737335-d610492a-df17-4787-af79-734afae134a6.png)

Thanks,
Surya
